modules/jupyter/variables.tf

```diff
@@ -47,7 +47,7 @@ variable "gcp_service_account_iam_roles" {
 variable "predefined_iam_roles" {
   description = "Predefined list of IAM roles to assign"
   type        = list(string)
-  default     = ["roles/compute.networkViewer", "roles/viewer", "roles/cloudsql.client", "roles/artifactregistry.reader", "roles/storage.admin", "roles/iam.serviceAccountAdmin", "roles/compute.loadBalancerServiceUser"]
+  default     = ["roles/compute.networkViewer", "roles/viewer", "roles/cloudsql.client", "roles/artifactregistry.reader", "roles/storage.admin", "roles/iam.serviceAccountAdmin", "roles/compute.loadBalancerServiceUser", "roles/iam.serviceAccountTokenCreator"]
```

Comment:
This is becoming a very powerful account with serviceAccountTokenCreator, serviceAccountAdmin, and storage.admin. Can we cut down these permissions? At a minimum, we need to document why each of these permissions are required as defaults.

Reply:
Add a todo to review all the permissions